scripts/create-users.sh

```diff
@@ -6,7 +6,7 @@ echo "Setting up required Users..."
 
 ./actions/create-user.sh "ccd-import@fake.hmcts.net" "CCD" "Import" "London01" "ccd-import" "[\"ccd-import\"]"
 ./actions/create-user.sh "${IA_SYSTEM_USERNAME}" "System" "user" "${IA_SYSTEM_PASSWORD}" "caseworker" "[\"caseworker\", \"caseworker-ia\", \"caseworker-ia-system\"]"
-./actions/create-user.sh "${WA_SYSTEM_USERNAME}" "WASystem" "WaUser" "${WA_SYSTEM_PASSWORD}" "caseworker" "[\"caseworker\", \"caseworker-ia\", \"caseworker-ia-system\"]"
+./actions/create-user.sh "${WA_SYSTEM_USERNAME}" "WASystem" "WaUser" "${WA_SYSTEM_PASSWORD}" "caseworker" "[\"caseworker\", \"caseworker-ia\", \"caseworker-ia-system\", \"caseworker-wa-task-configuration\"]"
```

Comment:
intially I created this to avoid using the IA one, but since we have a new role should we remove `caseworker-ia-system` from it ?

Reply:
I raised the following issue to the IA dev channel and Marcin got back to me. Basically, the issue if I remove that role then the ia-case-api does not work properly. I can confirm this behaviour because I reproduced many times locally with the hope to get to the bottom of it, but I couldn't. I can confirm we need that role for the submitAppeal event so that it triggers the notifications service. Without that role the notification is not triggered causing the event to fail.